src/style/_layout.scss

```diff
@@ -699,7 +699,6 @@ body
 .renderedMarkdown {
   p {
     @include text_body();
-    font-size: 14px;
```

Comment:
I wasn't 100% if the 14px was intended or not. Here's the difference:

<img width="569" alt="screen shot 2017-08-01 at 9 05 09 pm" src="https://user-images.githubusercontent.com/791222/28855925-2a0f6982-76fd-11e7-8eb8-e7625b0a1ec7.png">

<img width="566" alt="screen shot 2017-08-01 at 9 05 18 pm" src="https://user-images.githubusercontent.com/791222/28855926-2d681142-76fd-11e7-9458-c08da031d2e4.png">